lib/model/src/main/java/io/atlasmap/v2/Property.java

```diff
@@ -12,6 +12,8 @@
 
     protected FieldType fieldType;
 
+    protected String scope;
```

Comment:
Was it easier to put `scope` here as well? This `Property` is used to define a fallback value apart from `PropertyField`, and `scope` only makes sense on `PropertyField`.

Reply:
Needed to set the scope argument when editing an existing property.